### DISCUSSION TITLE
商业版oceanbase2.2.77如何解决oblogproxy的问题

### DISCUSSION THREAD

**Discussion by moy777** [apache/flink-cdc#3019] [Q&A]
@leonardBang 老师您好  我们在使用oceanbasecdc组件的时候遇到了一些问题 cdc版本为2.42.     oceanbase版本为商业版2.2.77，通过代码来看oblogproxy似乎是必须的组件，但是我们在oceanbase官网上只找到了开源版本的obproxy，且提工单找技术支持的答复是2.x版本不提供oblogproxy 想问一下为什么cdc的文档里面是支持2.x的，如果确实支持我们该怎么获取对应版本的oblogproxy
![image](https://github.com/ververica/flink-cdc-connectors/assets/48304802/5d07140d-215b-4d58-b78c-0e274648342d)

**Comment by whhe** [comment]
oblogproxy 企业版历史上支持过低版本的 ob 企业版，目前由于支持人力有限，已经不再对外提供，当前主要服务公有云用户，主推 binlog service （oblogproxy binlog 模式），对接 flink cdc mysql source connector。

**Reply by moy777** [reply]
那我们专有云的oceanbase目前没有升级计划，如果想要用到cdc的特性要怎么办呢

**Comment by whhe** [comment]
> 那我们专有云的oceanbase目前没有升级计划，如果想要用到cdc的特性要怎么办呢

flink cdc oceanbase source connector 应该是用不了了，可以咨询下 SA 有没有其他替代方案，比如 oms、dataworks 等等。
